### PR TITLE
WebRAVE VBET Curation

### PR DESCRIPTION
**Pull request by shelbysawyer**
Includes the necessary txt and json files

### CODE REVIEW

**Comment by shelbysawyer** [comment]
@MattReimer Hey, I'm not sure what's causing the issue here. I thought I had double-checked everything. Could it be something as simple as an extra bracket in one of the json files?

File: Symbology/web/VBET/vbet_active_channel.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ],
+    "layerStyles": [
+        {
+            "id": "vbet-active-channel-cd8gcb",
+            "type": "fill",
+            "source": "composite",
+            "source-layer": "vbet_active_channel-cd8gcb",
+            "layout": {},
+            "paint": {"fill-color": "hsl(206, 70%, 72%)", "fill-opacity": 0.7}
+        }
```
Comment (MattReimer) [inline Symbology/web/VBET/vbet_active_channel.json:14]: @shelbysawyer  The issue is that the square bracket doesn't terminate.

Once you get the brackets in the right order this shouldpass.

File: Symbology/web/VBET/vbet_active_floodplain.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        "id": "vbet-active-floodplain-b076i6",
+        "type": "fill",
+        "source": "composite",
+        "source-layer": "vbet_active_floodplain-b076i6",
+        "layout": {},
+        "paint": {"fill-color": "hsl(118, 58%, 69%)", "fill-opacity": 0.7}
+    }
+  ]
+}
```
Comment (shelbysawyer) [inline Symbology/web/VBET/vbet_active_floodplain.json:16]: @MattReimer Is this the same thing happening here? Do I just need to go through and get rid of this (and it looks like 4 others) curly bracket?

Comment (MattReimer) [reply Symbology/web/VBET/vbet_active_floodplain.json:16]: Vscode should find these problems for you so check the problems tab of the terminal to see if they check out. They should also show up as little red squiggles like a spellcheck if there's a problem

Comment (shelbysawyer) [reply Symbology/web/VBET/vbet_active_floodplain.json:16]: Okay, not seeing issues beyond that one in `vbet_active_channel` json. Hopefully I'm understanding correctly.

Comment (MattReimer) [reply Symbology/web/VBET/vbet_active_floodplain.json:16]: That just means that particular commit broke. Everything looks good to me.

**Review by MattReimer — APPROVED** [review]
👍

**Pull request merged**